lineoperations/src/linefunctions.c

```diff
@@ -295,8 +295,6 @@ void sortlines(GeanyDocument *doc, gboolean asc) {
 	total_num_chars = sci_get_length(doc->editor->sci);
```

Comment:
Good catch :) and if I was more careful yesterday I would have seen my compiler told me.  Updated.